pkg/metainfo/kvmetainfo/buckets_test.go

```diff
@@ -351,17 +351,17 @@ func newDB(planet *testplanet.Planet) (*DB, error) {
 		return nil, err
 	}
 
-	rs, err := eestream.NewRedundancyStrategy(eestream.NewRSScheme(fc, int(1*memory.KB)), 3, 4)
+	rs, err := eestream.NewRedundancyStrategy(eestream.NewRSScheme(fc, 1*memory.KB.Int()), 3, 4)
```

Comment:
Shouldn't all the `memory.KB` etc. now be `memory.KiB`?

Reply:
Yes, but let's fix them in another PR.